### PR TITLE
Ingestion: IJCLCLP 2022

### PR DESCRIPTION
**Pull request by anthology-assist**
## Checklist (for every volume)

1. [x] Make sure the branch is merged with the latest `master` branch
1. [x] Ensure that there are editors listed in the `<meta>` block
1. [x] If it's a workshop, add a `<venue>ws</venue>` tag
1. [x] Add events to their relevant SIGs
1. [x] Look at the venue listing for prior years, and ensure that the new volume titles are consistent. You can do this by clicking on the venue name from a paper page, which will take you to the vendor listing.
1. [x] Navigate to the event page preview (e.g., https://preview.aclanthology.org/icnlsp-ingestion/events/icnlsp-2021/), and page through, to see if there are any glaring mistakes
1. [x] Skim through the complete listing, looking for mis-parsed author names.
1. [x] Download the frontmatter and verify that the table of contents matches at least three randomly-selected papers
1. [x] Download 3–5 PDFs (including the first and last one) and make sure they are correct (title, authors, page numbers).

### CODE REVIEW

File: data/xml/2022.ijclclp.xml [hunk 1]
```diff
@@ -0,0 +1,139 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2022.ijclclp">
+  <volume id="1" ingest-date="2024-06-05" type="proceedings">
+    <meta>
+      <booktitle>International Journal of Computational Linguistics {\&amp;} {C}hinese Language Processing, Volume 27, Number 1, June 2022</booktitle>
```
Comment (mjpost) [inline data/xml/2022.ijclclp.xml:5]: stray latex characters

Comment (anthology-assist) [reply data/xml/2022.ijclclp.xml]: Done.

File: data/xml/2022.ijclclp.xml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+      <author><first>Ren-feng</first><last>Duann</last></author>
+      <author><first>Shu-I</first><last>Chiu</last></author>
+      <author><first>Hui-Wen</first><last>Liu</last></author>
+      <url hash="dfc0a2bd">2022.ijclclp-1.6</url>
+      <bibkey>duann-etal-2022-yi</bibkey>
+    </paper>
+  </volume>
+  <volume id="2" ingest-date="2024-06-05" type="proceedings">
+    <meta>
+      <booktitle>International Journal of Computational Linguistics &amp; {C}hinese Language Processing, Volume 27, Number 2, December 2022</booktitle>
```
Comment (mjpost) [inline data/xml/2022.ijclclp.xml:66]: Same LaTeX error

**Review by mjpost — APPROVED** [review]

**Comment by mjpost** [comment]
I fixed an inconsistency. FTR: image

**Pull request merged**